pkg/projects/Microsoft.NETCore.App/netcoreapp1.1/project.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Updated packages:
```
    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.4-beta-24403-04",
    "System.Diagnostics.Process": "4.1.1-beta-24403-05",
    "System.Linq.Expressions": "4.1.1-beta-24403-05",
    "System.Linq.Parallel": "4.0.2-beta-24403-05",
```

new additions:
```
    "Microsoft.NETCore.Targets": "1.0.3-beta-24403-05",
    "System.Runtime": "4.1.1-beta-24403-05",
```

The justification for adding Targets is that the native compression packages use the Targets import to determine which runtime-specific packages to include. When the stable compression package imports the stable Targets package, we end up lacking win10-arm64 assets as the last stable Targets package doesn't acknowledge that RID. Rather than modifying the clrcompression package (and potentially the other native packages for other new RIDS) and forcing updates of that version, I opted to instead have NetCore.App depend on a more recent Targets package.

It's somewhat tricky here because I don't want to update more versions than I need to and I don't want to break compat with netcoreapp1.0. I want netcoreapp1.0 and the associated packages to be as close to they are now as possible while still enabling netcoreapp1.1 to function and pull in the correct, updated assets.